ckanext/composite/templates/scheming/display_snippets/composite.html

```diff
@@ -3,7 +3,7 @@
 {% set name_list = h.composite_get_name_list(field.subfields) %}
 {% set composite_dict = h.composite_get_as_dict(data[field.field_name]) %}
 
-<dl>
+<dl class="cnra_composite_display_snippet_dl">
```

Comment:
We should not include mentions of `cnra` since the composite extension can be installed for any instance.
```suggestion
<dl class="composite-display-snippet">
```